src/golang/lib/collections/artifact/artifact_test.go

```diff
@@ -17,17 +15,15 @@ func TestSerializingAndDeserializingArtifact(t *testing.T) {
 	atf := artifact.DBArtifact{
 		Id:   id,
 		Name: "test",
-		Spec: *artifact.NewSpecFromTable(
-			table.Table{},
-		),
+		Type: artifact.TableType,
 	}
 
 	rawAtf, err := json.Marshal(atf)
 	require.Nil(t, err)
 
+	// TODO(cgwu): Temporarily commenting this out. Will revisit after finalizing the new type struct.
```

Comment:
Are you leaving this as is? Is there a linear task we can link here for more context.

Reply:
I already uncommented the code. Just forgot to remove the comment line...